index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sajjda Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <section>
+    <header id="main-header">
+      <div class="container">
+          <h2 id="logo">Sajjad</h2>
+          <img src="images/menu-icon.png" id="menu-icon" alt="menu icon">
```

Comment:
## OPTIONAL

- [ ] kindly note that the Umberger button should be in a `<nav><nav>` tag. As this will contain your navigation links in the future milestones.